README.md

```diff
@@ -73,6 +73,10 @@ stack setup
 stack install
 ```
 
+On Linux, the following libraries should be installed:
+- libexpat1-dev
```

Comment:
Is the expat library not automatically include with 
```
flags:
  hexpat:
    bundle: true
```
in stack.yaml (the linux variant).

Reply:
Good point. See the new changes I pushed.